tests/datamodules/test_inria.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I'm trying to get rid of the tests in `tests/datamodules` because they aren't actually that useful. Instead, I'm trying to test things directly in `tests/trainers` with real pytorch-lightning trainers.

Reply:
I've added a test config for the dataset but I would like to keep this test for now. Going to need this in the prediction PR.